auth0_flutter_platform_interface/test/auth0_flutter_web_platform_test.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:auth0_flutter_platform_interface/src/account.dart';
+import 'package:auth0_flutter_platform_interface/src/auth0_flutter_web_platform.dart';
+import 'package:auth0_flutter_platform_interface/src/user_agent.dart';
+import 'package:auth0_flutter_platform_interface/src/web/client_options.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class TestAuth0FlutterWeb extends Auth0FlutterWebPlatform {
```

Comment:
This file should fix the test coverage for `Auth0FlutterWebPlatform`.
I ran `flutter test --coverage` before adding these tests and the code coverage lcov.info didn't list anything for `Auth0FlutterWebPlatform`.

After adding these tests and running `flutter test --coverage` again, the lcov.info showed the class as fully tested.

Reply:
Thanks for adding this.